src/components/App.js

```diff
@@ -31,11 +31,24 @@ class App extends Component {
     this.setState({ currentUser: user })
   }
 
+  logout () {
```

Comment:
I think we should also call `data.setUserToken(null)` here and clear the token from `window.localStorage`.

Reply:
I have added the line requested above and pushed it to the remote branch. Do I need to add more to remove it from local storage? I have `window.localStorage.clear()` in the logout method and the local storage appears to be clear when I log it in the console.